JSDemos/Demos/Charts/LoadDataOnDemand/description.md

```diff
@@ -1 +1,29 @@
-In&nbsp;this demo, the chart loads data as&nbsp;you pan&nbsp;it. Once a&nbsp;data block is&nbsp;loaded, it&nbsp;stays in&nbsp;memory to&nbsp;reduce requests to&nbsp;the server.
\ No newline at end of file
+In this demo, the [range area](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/RangeArea/) Chart loads data as you pan it. Once the component loads a data block, this data stays in memory to reduce requests to the server. To implement this functionality, do the following:
+
+1. Configure a data source.
+
+    - Assign a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) with an empty [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/) to the Chart [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property.
+
+    - Set the [sort](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#sort) property to `date`.
+
+    - Disable the [paginate](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#paginate) property to prevent data from partitioning.
+
+2. Configure the Chart to support the data load on demand.
+
+    - Set the [visualRangeUpdateMode](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#visualRangeUpdateMode) property to `keep`.
+
+    - Specify a [loadingIndicator](/Documentation/ApiReference/UI_Components/dxChart/Configuration/loadingIndicator/).
+
+    - Specify the initial [visualRange](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/visualRange/).
+    
+    - Implement a function that computes the Chart current [visualRange](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/visualRange/). In this function, call a method that checks whether a user changed the visual range.
+
+3. Implement functions that load the data.
+
+    - The Chart argument data is days. If users pan the Chart left or right more than half a day, the data starts to load. The `onVisualRangeChanged` method checks this condition, and if the condition is true, the method calls the uploading function.
+
+    - The `getDataFrame` Ajax request gets the new data frame from the server.
+
+    - The `uploadDataByVisualRange` function takes two arguments: current visual range and the component's instance. This function gets the starting and ending points of the visual range and the bounds of already loaded data and calls the `getDataFrame` function. Then, the `uploadDataByVisualRange` function reloads the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) and updates the visual range.
```

Comment:
I would put this item above `getDataFrame`. It continues the narrative of how the control gets data updates. We start with user operation, then we talk about how half a day is enough to trigger an update... so now we need to talk about how an update is actually implemented. If we put `getDataFrame` in the middle, it breaks the narrative. 

```suggestion
    - The `uploadDataByVisualRange` function analyzes the starting and ending points of the visual range and the bounds of already loaded data. If necessary, it calls the `getDataFrame` function to obtain new data points. Finally, it reloads the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) and saves the current visual range.
```